orc8r/cloud/go/services/orchestrator/obsidian/models/defaults.go

```diff
@@ -44,6 +44,13 @@ func NewDefaultSentryConfig() *NetworkSentryConfig {
 	}
 }
 
+func NewDefaultStateConfig() *NetworkStateConfig {
+	syncInterval := uint32(60)
```

Comment:
Part of the task includes actually increasing this default value

Also nit: here and elsewhere, use the swag.uint32 pattern I mention above

Reply:
bumped it to 5 mins